addon/locales/en/translations.js

```diff
@@ -46,31 +46,42 @@ export default {
             buttonSubmit: 'Create account'
         },
         navbar: {
+            osf: 'OSF',
+            openScienceFramework: 'Open Science Framework',
             add: 'Add',
-            addAPreprint: 'Add a {{preprintWords.preprint}}',
+            addAPreprint: 'Add a {{preprintWords.Preprint}}',
             browse: 'Browse',
             cancelSearch: 'Cancel search',
+            dashboard: 'Dashboard',
             donate: 'Donate',
             goHome: 'Go home',
+            meetings: 'Meetings',
             myProjects: 'My Projects',
             myQuickFiles: 'My Quick Files',
+            newProjects: 'New Projects',
+            preprints: 'Preprints',
+            registries: 'Registries',
             reviews: 'My Reviewing',
             search: 'Search',
             searchHelp: 'Search help',
+            altSearchOSF: 'Search OSF',
             searchTheOSF: 'Search the OSF',
             sendSearch: 'Send search query',
             support: 'Support',
+            toggleNavigation: 'Toggle navigation',
             togglePrimary: 'Toggle primary navigation',
             toggleSecondary: 'Toggle secondary navigation',
         },
         authDropdown: {
-            logOut: 'Log out',
+            logOut: 'Log Out',
```

Comment:
This differs from osf.io, but I assume this is the correct version, right?

Reply:
Yup, all of the text in the navbar should be title-case.  The OSF side of it should also have the capitalized `Out` too.  I can definitely make a separate ticket/PR for the osf.io navbar to make the capitalization consistent across both sides.